frontend/apps/core-app/package.json

```diff
@@ -54,5 +65,25 @@
     "collectCoverageFrom": [
       "src/**/*.{js,jsx,ts,tsx}"
     ]
+  },
+  "expo-yarn-workspaces": {
+    "symlinks": [
+      "@unimodules/core",
```

Comment:
what are the criteria for a package to be listed here?

Reply:
Good catch, I copied this from an example. I need to dig into this a bit more

Reply:
The docs on this are [here](https://github.com/expo/expo/tree/master/packages/expo-yarn-workspaces#symlinks). What I'd say is if a build fails and it complains about not being able to find a package or an issue with a package, then we should try adding it in here. It's not clear what should be here without digging into it's source code.

I have cleaned up the list though.

Reply:
yeah, I had seen that, didn't find it very explanatory though. 
what made you choose the ones to keep/remove?

Reply:
Kept the ones that are in our package.json (assuming they were in the example snippet for a reason), then did trial and error on the others.

I think what the docs mean is that some packages will look for a file in the path` ./node_modules/<package>/file.ext`. With yarn workspaces these files live at `../../../node_modules/<package>/file.ext`. If it searches for the file where it expects it then it will get an error. Adding the package to the symlink section sets up a symlink from one node_modules/<package> to the other. I think this issue is specific to react native because some of files will be android/ios files.